# Discrete Probability Distributions

## Objectives ##
- Identify and Construct discrete probability distributions.
- Find the expected value of discrete probability distributions.
- Find the standard deviation of discrete probability distributions.

## Discrete Probability Distributions
A **probability distribution** is a table, formula, or rule that gives the probability for each outcome of an experiment. When the possible outcomes of an experiment only take on discrete values (that is, the outcomes can take on only certain values and not the values in between), we say the experiment has a **discrete probability distribution**. Almost all of the experiments we have studied in this text so far have **discrete probability distributions**. 

For example, the probability distribution of rolling a six-sided die is described by {numref}`pd-die`. Note that the outcomes of rolling a die can only take on certain discrete value (specifically, the numbers $1$, $2$, $3$, $4$, $5$, and $6$), but not the values in between (like $2.457$).

```{list-table} The discrete probability distribution for rolling a fair six-sided die.
:header-rows: 1
:name: pd-die
* - $X$
  - $P(X)$
* - $1$
  - $\dfrac{1}{6} = 0.1667$
* - $2$
  - $\dfrac{1}{6} = 0.1667$
* - $3$
  - $\dfrac{1}{6} = 0.1667$
* - $4$
  - $\dfrac{1}{6} = 0.1667$
* - $5$
  - $\dfrac{1}{6} = 0.1667$
* - $6$
  - $\dfrac{1}{6} = 0.1667$
```

In contrast, if the outcome of an experiment can take on take on the value of any fraction, decimal, or irrational number within the range of allowed values, we say the experiment has a **continuous probability distribution**. For example, if we conduct an experiment where we measure the height of a randomly chosen person, the outcome could be that the individual is $64.41$ inches tall, or maybe $72.683$ inches tall, or it could be any decimal value in between. We will learn more about continuous probability distributions later.

A **discrete probability distribution** has two characteristics:

1. Since each outcome can't have less than a $0\%$ chance of happening or more than a $100\%$ chance of happening, the probability of each outcome must be between $0$ and $1$.
2. Since one of the outcomes is guaranteed to happen, the sum of the probabilities of all the outcomes must be $1$.

***


### Example 3.5.1 ###
A child psychologist is interested in the number of times a newborn baby's crying wakes its mother after midnight. For a random sample of $50$ mothers, the information in {numref}`pd-cry` was obtained. Does {numref}`pd-cry` represent a discrete probability distribution?

```{list-table} Relative frequency table for the number of times a newborn baby's cry wakes a sample of $50$ mothers after midnight.
:header-rows: 1
:name: pd-cry
* - Number of times cry wakes mother after midnight
  - Relative Frequency
* - $0$
  - $\dfrac{2}{50} = 0.04$
* - $1$
  - $\dfrac{11}{50} = 0.22$
* - $2$
  - $\dfrac{23}{50} = 0.46$
* - $3$
  - $\dfrac{9}{50} = 0.18$
* - $4$
  - $\dfrac{4}{50} = 0.08$
* - $5$
  - $\dfrac{1}{50} = 0.02$
```

#### Solution

{numref}`pd-cry` represents a discete probability distribution because:

1. Each outcome has a probability between $0$ and $1$.
2. The sum of the probabilities of all the outcomes is $1$:

$$ \frac{2}{50} + \frac{11}{50} + \frac{23}{50} + \frac{9}{50} + \frac{4}{50} + \frac{1}{50} = 1 $$

This means that if we select one of the $50$ mothers at random, the table tells us the probability of each possible outcome. For example, the table tells us that the probability that the selected mother was woken up twice after midnight by her crying baby is $0.46$.

***


### Example 3.5.2 ###
Suppose Nancy has classes **three days** a week. She attends classes **three days a week 80%** of the time, **two days 15%** of the time, **one day 4%** of the time, and **no days 1%** of the time. Suppose one week is randomly selected.

1. What is $X$?
2. $X$ can take on what values?
3. Construct a probability distribution table like the one in example 4.1. The table should have two columns labeled $X$ and $P(X)$. What does the $P(X)$ column sum up to?

#### Solution ####
##### Part 1 #####
$X =$ the number of days Nancy attends class.

##### Part 2 #####
$X$ can take on the values $0$, $1$, $2$, or $3$.

##### Part 3 #####
```{list-table} A discrete probability distribution for the number of days of the week that Nancy attends class.
:header-rows: 1
:name: pd-class
* - $X$
  - $P(X)$
* - $0$
  - $0.01$
* - $1$
  - $0.04$
* - $2$
  - $0.15$
* - $3$
  - $0.80$
```

The $P(X)$ column sums  up to:

$$0.01 + 0.04 + 0.15 + 0.80 = 1.00.$$

This is expected. If the probabilities did not add up to one, {numref}`pd-class` wouldn't be a discrete probability distribution.

***

## Expected Value ##
The **expected value** is often referred to as the **"long-term" average** or **mean**. This means if you repeat an experiment many times, over the long term you would **expect** this average.

Recall that the **Law of Large Numbers** states that as the number of trials in a probability experiment increases, the theoretical probability of an event and the proportion of experiments where that event occurs gets closer and closer. 

For example, when you roll a six-sided die, probabiliy says you should expect each face of the die to come up in $\frac{1}{6}$ of the rolls. But if you only roll the die $6$ times, we wouldn't be shocked if you rolled $2$ fives (meaning you rolled a five in $\frac{2}{6}$ of the rolls). However, if you rolled the die $6{,}000{,}000$ times, we would expect you to roll a five in very nearly $\frac{1}{6}$ of the rolls. As the number of die rolls increases, the proportion of fives that you roll tends to get closer and closer to the probability of rolling a five.

When evaluating the long-term results of statistical experiments, we often want to know the “average” outcome. This “long-term average” is known as the **mean** or **expected value** of the experiment and is denoted by the Greek letter $\mu$. In other words, after conducting many trials of an experiment, you would expect this average value.

The expected value of a discrete probability distribution function can be found using the formula

$$ \mu = \sum x \cdot P(x). $$

In words, we multiply each value $x$ by the probability $P(x)$ that the value will occur. Then we add up the products $x \cdot P(x)$.

To illustrate this idea, again return to the idea of rolling a six-sided die. The possible outcomes of rolling a die are $x = 1, 2, 3, 4, 5, 6$, and the probability of each outcome is $\frac{1}{6}$. Then the expected value is

$$
\begin{align}
\mu &= \sum x \cdot P(x) \\
&= 1\cdot P(1) + 2 \cdot P(2) + 3 \cdot P(3) + 4 \cdot P(4) + 5 \cdot P(5) + 6 \cdot P(6) \\
&= 1\cdot \frac{1}{6} + 2 \cdot \frac{1}{6} + 3 \cdot \frac{1}{6} + 4 \cdot \frac{1}{6} + 5 \cdot \frac{1}{6} + 6 \cdot \frac{1}{6} \\
&= 3.5,
\end{align}
$$

meaning if we rolled the die many times, we would expect the average of all the rolls to be close to $3.5$.

We can use R to do this same calculation more quickly.

In [1]:
x = c(1, 2, 3, 4, 5, 6)
Px = c(1/6, 1/6, 1/6, 1/6, 1/6, 1/6)

mu = sum(x * Px)
mu

[1] 3.5

***


### Example 3.5.3 ###
A men's soccer team plays soccer zero, one, or two days a week. The probability that they play zero days is $0.2$, the probability that they play one day is $0.5$, and the probability that they play two days is $0.3$. On average, how many games per week would we expect the team to play next year?

#### Solution ####
We need to calculate the expected value, which estimates the average number of games the team plays per week this next year. The discrete probability distribution is given in {numref}`pd-soccer`, where $X$ is the number of games played per week.

```{list-table} A discrete probability distribution for the number of days the soccer team plays in a week.
:header-rows: 1
:name: pd-soccer
* - $X$
  - $P(X)$
* - $0$
  - $0.2$
* - $1$
  - $0.5$
* - $2$
  - $0.3$
```

We can use R to find the expected value.

In [1]:
x = c(0, 1, 2)
Px = c(0.2, 0.5, 0.3)

mu = sum(x*Px)
mu

[1] 1.1

The mean or expected value of the distribution is $\mu = 1.1$ games per week. That is, we expect the soccer team to play an average of $1.1$ games per week next year.

***


### Example 3.5.4 ###
Suppose you play a game of chance in which five numbers are randomly chosen from $0$ to $9$ by a computer. If you match all five numbers in order, you win $\$100,000$. If you lose, you pay $\$2$. Over the long term, what is your expected profit per game?

#### Solution ####
We need to calculate the expected value because it will tell us how much we expect to profit per game on average.

Let's first construct a table of the discrete probability distribution. Let $X$ be the amount of money you profit from playing the game. The values that $X$ can take on are $x = 100{,}000$ and $x = -2$. (Note, since we are ultimately interested in the profit, $X$ does not take on the values of the randomly chosen numbers.)

Now let's look at the probability of winning the game. Let
- $N_1 =$ the event you pick the right 1st number
- $N_2 =$ the event you pick the right 2nd number
- $N_3 =$ the event you pick the right 3rd number
- $N_4 =$ the event you pick the right 4th number
- $N_5 =$ the event you pick the right 5th number

The probability that you win is

$$ P(x = 100,000) = P(N_1\text{ AND }N_2\text{ AND }N_3\text{ AND }N_4\text{ AND }N_5) $$

Note that since the numbers are chosen *with replacement*, $N_1$, $N_2$, $N_3$, $N_4$, and $N_5$ are all independent events. Thus

$$\begin{align}
P(N_1\text{ AND }N_2\text{ AND }N_3\text{ AND }N_4\text{ AND }N_5) 
&= P(N_1) \cdot P(N_2) \cdot P(N_3) \cdot P(N_4) \cdot P(N_5) \\
&= \frac{1}{10} \cdot \frac{1}{10} \cdot \frac{1}{10} \cdot \frac{1}{10} \cdot \frac{1}{10} \\
&= \frac{1}{10^5} \\
&= 0.00001
\end{align}$$

So the probability that you win is $P(100{,}000) = 0.00001$. Since you can only win or lose at this game (the game can't end in a draw), the probability that you lose is

$$ P(-2) = 1 - P(100{,}000) = 1 - 0.00001 = 0.99999. $$

{numref}`pd-game` describes the probability distribution for this game.

```{list-table} A discrete probability distribution for the amount of money you win or lose playing the game.
:header-rows: 1
:name: pd-game
* - $X$
  - $P(X)$
* - $100{,}000$
  - $0.00001$
* - $-2$
  - $0.99999$
```

Now use R to calculate the expected value.

In [3]:
x = c(100000, -2)
Px = c(0.00001, 0.99999)

mu = sum(x*Px)
mu

[1] -0.99998

The expected value is $\mu = -\$0.99998$. Since the expected value is negative, this means that if you played this game many times, you would lose an average of almost \$1 per game.

***

## Standard Deviation ##
The standard deviation $\sigma$ of a discrete probability distribution is

$$ \sigma = \sqrt{\sum\left[(x - \mu)^2 P(x)\right]}, $$

where $\mu$ is the expected value of the distribution. The standard deviation tells us the standard (or average) amount the outcome of an experiment deviates from the expected value.

***


### Example 3.5.5 ###
Patients who have had an appendectomy must stay in the hospital for $1$, $2$, or $3$ days. Patients stay in the hospital for $1$ day $40\%$ of the time, for $2$ days $55\%$ of the time, and for $3$ days $5\%$ of the time. 

1. How long should a patient expect to stay in the hospital? 
2. What is the standard deviation of the distribution?

#### Solution
Let's first construct the probability distribution. Let $X$ be the number of days the patient stays in the hosplital. The possible values of $X$ are $x = 1, 2, 3$. {numref}`pd-appendectomy` is the probability distribution for the number of days appendectomy patients stay in the hospital after surgery.

```{list-table} A discrete probability distribution for the number of days appendectomy patients stay in the hospital after surgery.
:header-rows: 1
:name: pd-appendectomy
* - $X$
  - $P(X)$
* - $1$
  - $0.40$
* - $2$
  - $0.55$
* - $3$
  - $0.05$
```

##### Part 1
The expected value gives the best estimate for the number of days a patient should expect to stay in the hospital.

In [4]:
x = c(1, 2, 3)
Px = c(0.40, 0.55, 0.05)

mu = sum(x*Px)
mu

[1] 1.65

The expected value is $\mu = 1.65$. A patient should expect to spend, on average, $1.65$ days in the hospital after an appendectomy.

##### Part 2 
We can use R to calculate the standard deviation.

In [6]:
sigma = sqrt( sum( (x - mu)^2 * Px ) )
sigma

[1] 0.5722762

So the standard deviation is $\sigma = 0.572$. This means it is not uncommon for a patient to spend $0.572$ days less or $0.572$ days more than the expected $1.65$ days in the hospital after surgery

***


### Example 3.5.6 ###
A hospital researcher is interested in the number of times the average post-op patient will ring the nurse during a $12$-hour shift. For a random sample of $50$ patients, {numref}`pd-nurse` was obtained.

```{list-table} A relative frequency table for the number of times $50$ patients rings the nurse in a $12$-hour shift.
:header-rows: 1
:name: pd-nurse
* - Number of Times Patient Rings Nurse
  - Relative Frequency
* - $0$
  - $\frac{4}{50}$
* - $1$
  - $\frac{8}{50}$
* - $2$
  - $\frac{16}{50}$
* - $3$
  - $\frac{14}{50}$
* - $4$
  - $\frac{6}{50}$
* - $5$
  - $\frac{2}{50}$
```

1. Find the mean.
2. Find the standard deviation.

#### Solution
First, note that {numref}`pd-nurse` is a discrete probability distribution, where $X$ is the number of times the patient rings the nurse, and $P(X)$ is the relative frequency.

##### Part 1
Remember, for a probability distribution, the mean and the expected value are the same thing.

In [8]:
x = c(0, 1, 2, 3, 4, 5)
Px = c(4/50, 8/50, 16/50, 14/50, 6/50, 2/50)

mu = sum(x*Px)
mu

[1] 2.32

The mean or expected value is $\mu = 2.32$. This means, on average, we expect a patient to ring the nurse $2.32$ times during a $12$-hour shift.

##### Part 2

In [9]:
sigma = sqrt( sum( (x - mu)^2 * Px ) )
sigma

[1] 1.223765

The standard deviation is $\sigma = 1.2238$. This means that it is common for a patient to ring the nurse $1.2238$ times less or $1.2238$ times more than the expected $2.32$ rings.